test/controllers/SecretControllerSpecGetAPITest.scala

```diff
@@ -0,0 +1,27 @@
+package controllers
```

Comment:
`/SecretControllerSpecGetAPITest.scala ` Avoid using such test class names.

Writting test class for each endpoint is not good idea.

Class test name can be `/SecretControllerTest.scala`

Reply:
We need to cover  positive as well as negative test cases as well, right ?

Reply:
yes